docs/administrator/security/cert-message.md

```diff
@@ -0,0 +1,665 @@
+        Subject Public Key Info:
+        X509v3 extensions:
+            X509v3 Key Usage: critical
+                Digital Signature, Key Encipherment
+            X509v3 Extended Key Usage:
+                TLS Web Client Authentication, TLS Web Server Authentication
+            X509v3 Basic Constraints: critical
+                CA:FALSE
+```
+
+## Karmada Search ETCD Client Certificate
+
+### Introduction:
+
+The Karmada Search ETCD client certificate is used for identity authentication when `karmada-search` accesses the ETCD data store. It allows `karmada-search` to securely retrieve resource information from ETCD while ensuring proper control of data access permissions.
+
+### Content Example:
+
+```
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            5a:f6:2d:21:a5:31:92:99:32:07:00:dd:8f:37:0a:c0:ae:02:2f:c1
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN=karmada
+        Validity
+            Not Before: %b %d %H:%M:%S %Y GMT
+            Not After : %b %d %H:%M:%S %Y GMT
+        Subject: O=system:masters, CN=system:karmada:karmada-search-etcd-client
+        Subject Public Key Info:
+        X509v3 extensions:
+            X509v3 Key Usage: critical
+                Digital Signature, Key Encipherment
+            X509v3 Extended Key Usage:
+                TLS Web Client Authentication, TLS Web Server Authentication
+            X509v3 Basic Constraints: critical
+                CA:FALSE
+```
+
+## ETCD ETCD Client Certificate
+
+### Introduction:
+
+The ETCD ETCD client certificate is a key security credential for the Karmada ETCD service's self-health check and internal communication, ensuring the security and availability of the ETCD service.
+
+### Content Example:
+
+```
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            5a:f6:2d:21:a5:31:92:99:32:07:00:dd:8f:37:0a:c0:ae:02:2f:c1
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: CN=karmada
+        Validity
+            Not Before: %b %d %H:%M:%S %Y GMT
+            Not After : %b %d %H:%M:%S %Y GMT
+        Subject: CN=system:karmada:etcd-etcd-client
+        Subject Public Key Info:
+        X509v3 extensions:
+            X509v3 Key Usage: critical
+                Digital Signature, Key Encipherment
+            X509v3 Extended Key Usage:
+                TLS Web Client Authentication, TLS Web Server Authentication
+            X509v3 Basic Constraints: critical
+                CA:FALSE
+```
+
+## Karmada Scheduler GRPC Certificate
+
+### Introduction:
+
+The Karmada Scheduler GRPC certificate is used to encrypt and authenticate gRPC communication between `karmada-scheduler` and `karmada-scheduler-estimator`. `karmada-scheduler` uses this channel to obtain cluster load estimation information for scheduling decisions. This certificate ensures the secure transmission of these critical performance data.
+
+### Content Example:
```

Comment:
```suggestion
Here is an example of the Karmada Scheduler GRPC certificate:
```
The repeated use of the heading “### Content Example:” (appearing 10 times) comes across as somewhat simplistic or unrefined.